ezomero/_gets.py

```diff
@@ -1077,6 +1084,43 @@ def get_pyramid_levels(conn, image_id, across_groups=True):
     return levels
 
 
+@do_across_groups
+def get_table(conn, file_ann_id, across_groups=True):
+    """Get a table from its FileAnnotation object.
+
+    Parameters
+    ----------
+    conn : ``omero.gateway.BlitzGateway`` object
+        OMERO connection.
+    file_ann_id : int
+        ID of FileAnnotation table to get.
+    across_groups : bool, optional
+        Defines cross-group behavior of function - set to
+        ``False`` to disable it.
+
+    Returns
+    -------
+    table : object
+        Object containing the actual table. It can be either a list of
+        row-lists or a pandas Dataframe in case the optional pandas dependency
+        was installed.
+
+    Examples
+    --------
+    >>> table = get_table(conn, 62)
+    >>> print(table[0])
+    ['ID', 'X', 'Y']
+    """
+    if type(file_ann_id) is not int:
+        raise TypeError('File annotation ID must be an integer')
+    ann = conn.getObject('FileAnnotation', file_ann_id)
+    orig_table_file = conn.getObject('OriginalFile', ann.getFile().id)
+    resources = conn.c.sf.sharedResources()
+    table_obj = resources.openTable(orig_table_file._obj)
```

Comment:
do we need to catch whether someone gives a file ID that is not a table?

Reply:
I think we should - good catch. I'll try and figure out how to do that.